<?xml version="1.0"?>
<launch>
  <!-- tf frame들  on -->
    <!-- <node pkg="tf" type="static_transform_publisher" name="mobile_base"
    args="0.0 0.0 0.0 0 0 0 1 map mobile_base 10" />

    <node pkg="tf" type="static_transform_publisher" name="panda_base"
    args="0.30861 0.0 0.4405 0 0 0 1 mobile_base panda_base 10" />

    <node pkg="tf" type="static_transform_publisher" name="camera_base"
    args="-0.21855 -0.1159 1.5190 0 0 0 1 mobile_base camera_base 10" /> 

    <node pkg="tf" type="static_transform_publisher" name="rgb_optical_frame"
    args="-0.0039076 -0.0320149 -0.000327477 -0.49927 0.501692 -0.500734 0.498297 camera_base rgb_optical_frame 10" />  -->
    
    <!-- <include file="$(find gpd_ros)/launch/ur5.launch">
    </include> -->


    <param name="franka_arm_description" command="$(find xacro)/xacro '$(find franka_description)/robots/panda_arm_hand.urdf.xacro'"/>
    <param name="urdf_param" value="/franka_arm_description"/> 
  <node name="interfaceServer" pkg="base_placement_planner" type="interfaceServer" output="screen">
    <param name="yaml_path" type="string" value="$(find arm_motion_planner)/config/hw_param.yaml"/>
    <param name="ws_path" type="string" value="$(find base_placement_planner)/mapConfig/"/> 
    <param name="file_name" type="string" value="reachableDataAcc.bin"/>  <!-- 저장하거나 로드할 파일 이름 입력!!!!!!! -->
  </node>
<!-- required="true" -->
  <!-- Convert it into a point cloud ///  depth image proc PKG -->  
  <!-- <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />
    manipualbility :FrankaPanda  
    Reachability : justReachableAcc
  <node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/point_cloud_xyzrgb nodelet_manager">
    <remap from="depth_registered/image_rect" to="/k4a/depth/image_raw"/>
    <remap from="depth_registered/points" to="camera/depth_registered/points"/>
    <remap from="rgb/image_rect_color" to="/k4a/rgb/image_rect_color"/>
    <remap from="rgb/camera_info" to="/k4a/depth/camera_info"/>
  </node> -->
</launch>